harness/determined/pytorch/_pytorch_context.py

```diff
@@ -426,6 +442,24 @@ def _step_optimizer(
         is performed by the Determined training loop by setting the experiment configuration
         optimization.aggregation_frequency.
 
+        Here is a code example of using lambda function:
+
+        .. code-block:: python
+
+            self.context.step_optimizer(
+                self.opt1,
+                clip_grads=lambda params: torch.nn.utils.clip_grad_norm_(params, 0.0001),
+            )
+
+        Or you can use a function defined by ``def``:
+
+        .. code-block:: python
+
+            def clip_grads(params):
+                torch.nn.utils.clip_grad_norm_(params, 0.0001),
+
+            self.context.step_optimizer(self.opt1, clip_grads)
+
```

Comment:
Question: do we need to explain how to pass functions to users?  Like could we just say "here's a code example" and give them the `def` version, and let users who know about `lambda` use lambda on their own?